specification/professionalservice/resource-manager/readme.az.md

```diff
@@ -0,0 +1,28 @@
+## AZ
```

Comment:
For new service,  readme.az.md and readme.cli.md are not needed. Please remove those files.  Thanks

Reply:
Removed